tests/python/frontend/keras/test_forward.py

```diff
@@ -182,6 +182,7 @@ def test_forward_activations(self, keras):
             x = act_func(data)
             keras_model = keras.models.Model(data, x)
             verify_keras_frontend(keras_model)
+            verify_keras_frontend(keras_model, False, 'NHWC')
```

Comment:
minor suggestion: `verify_keras_frontend` will automatically set `need_transpose` to `false` when the layout is not `NCHW`. So in this case if you want to make it explicit, I suggest naming the parameters.

```
verify_keras_frontend(keras_model, need_transpose=False, layout='NHWC')
```

Reply:
OK, that's better.